compiler/circle-opselector/src/ModuleIO.cpp

```diff
@@ -27,7 +27,7 @@
 namespace opselector
 {
 
-std::unique_ptr<luci::Module> getModule(std::string &input_path)
+std::vector<char> getModelData(std::string &input_path)
```

Comment:
This PR seems to have multiple purpose of changes.
It would be better to split them. This one changed function prototype.